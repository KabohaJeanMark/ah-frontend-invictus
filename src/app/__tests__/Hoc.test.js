import React from 'react';
import toJson from 'enzyme-to-json'; // for snapshotting the component
import { shallow } from 'enzyme';
import Aux from '../../app/hoc/Aux';

describe('<Aux />', () => {
  let wrapper;

  it('renders without crashing', () => {
    wrapper = shallow(<Aux />);
  });

  it('matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
