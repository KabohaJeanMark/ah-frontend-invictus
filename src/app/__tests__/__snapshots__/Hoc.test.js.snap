// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Aux /> matches snapshot 1`] = `""`;
